plugins/scaffolder-backend-module-github/src/actions/githubPullRequest.ts

```diff
@@ -328,6 +344,10 @@ export const createPublishGithubPullRequestAction = (
             {
               files,
               commit: commitMessage ?? title,
+              author: {
```

Comment:
hm is it better to only give this argument here, if the name and/or email had been set? i just want to retain the old default behavior from before this new feature was added

Reply:
Done: https://github.com/backstage/backstage/pull/24628/commits/a35ef27eadeda1b47ded934c87709200254e30ce